components/Nav/Nav.tsx

```diff
@@ -55,6 +58,18 @@ const Nav = ({
 
   const hasNotifications = !!count && count > 0;
 
+  const handleSignInPageNavigation = () => {
```

Comment:
Nice! This is very much a minor issue but it annoyed me and means we need to add guards in E2E tests. This will remove that need

Reply:
Thanks, I am happy to help :)

Helping contributors to solve these minor but time consuming issues is my goal for this Hacktoberfest.